### PR TITLE
Design About Page

### PR DESCRIPTION
**Pull request by AmberPM**

### CODE REVIEW

File: src/routes/about/+page.svelte [hunk 1]
```diff
@@ -1,41 +1,328 @@
 <svelte:head>
 	<title>About Us</title>
 	<meta name="description" content="About this app" />
+	<link rel="preconnect" href="https://fonts.googleapis.com" />
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+	<link
+		href="https://fonts.googleapis.com/css2?family=Pixelify+Sans:wght@400..700&display=swap"
+		rel="stylesheet"
+	/>
+	<link rel="preconnect" href="https://fonts.googleapis.com" />
```
Comment (jcreek) [inline src/routes/about/+page.svelte:10]: These duplicates can be removed

File: src/routes/about/+page.svelte [hunk 2]
```diff
@@ -1,41 +1,328 @@
...
-		Share insights into the development process, challenges faced, and any interesting anecdotes or
-		milestones achieved during the project's journey.
-	</div>
+		<p class="font-semibold text-xl mt-20 mb-5 ml-5">Features</p>
 
-	<div id="contact-information">
-		Offer ways for users to get in touch with the development team, such as through social media, or
-		a contact form.
-	</div>
+		<!-- Ask Josh to help you with JavaScript so the card features can swipe -->
```
Comment (jcreek) [inline src/routes/about/+page.svelte:44]: Replace this comment with a TODO comment, e.g. `TODO (#41) - Make this a card carousel`

File: src/routes/about/+page.svelte [hunk 3]
```diff
@@ -1,41 +1,328 @@
...
+		<div class="card md:card-side bg-base-00 shadow-m">
+			<figure>
+				<img src="/OIG3.jpeg" alt="Album" />
```
Comment (jcreek) [inline src/routes/about/+page.svelte:47]: Should this have its size specified?

File: src/routes/about/+page.svelte [hunk 4]
```diff
@@ -1,41 +1,328 @@
...
-		Provide necessary legal disclaimers, privacy policies, or terms of service to ensure compliance
-		with relevant regulations and protect user rights.
+		<div id="legal-information">
+			<p class="font-semibold text-xl mt-0 mb-5 ml-5 text-white pt-10">Legal</p>
+
+			<p class="text-sm mb-5 ml-5 mr-5">
+				Living Dex Tracker is a fan-made website dedicated to providing information about Pokemon
+				creatures. We do not claim ownership of any Pokemon characters, images, or other content
+				featured on this website.
+				<br /><br />We do not generate revenue from the use of Pokemon content. Any advertisements
```
Comment (jcreek) [inline src/routes/about/+page.svelte:206]: Please can this be a second paragraph tag, rather than having two `br` tags to split the paragraphs.

**Review by jcreek — CHANGES_REQUESTED** [review]

File: src/routes/about/+page.svelte (hunk 3 above)
Comment (AmberPM) [reply src/routes/about/+page.svelte:44]: No, because it is a placeholder until the carousel is done.

**Review by jcreek — APPROVED** [review]

**Pull request merged**